plugins/hydra_ax_sweeper/tests/test_example_sweeper_plugin.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import importlib
+import math
+from typing import Any, Callable, List, Optional
+
+import pytest
+from omegaconf import DictConfig
+
+from hydra.core.plugins import Plugins
+from hydra.plugins.sweeper import Sweeper
+
+# noinspection PyUnresolvedReferences
+from hydra.test_utils.test_utils import TSweepRunner  # noqa: F401
+from hydra.test_utils.test_utils import SweepTaskFunction
```

Comment:
What is SweepTaskFunction? please consult me before adding core abstractions.

Reply:
I didnot add it. https://github.com/facebookresearch/hydra/blob/1888c7fba77f9e49b0b218c2294b1b0a5974482e/hydra/test_utils/test_utils.py#L178